DataScience/DashboardMpi/helpers/environment.py

```diff
@@ -1,13 +1,25 @@
-class environment:
-    def __init__(self, vw_path, runtime, job_pool, cache_path_gen=None,
-                 model_path_gen=None, pred_path_gen=None, cache_provider=None,
-                 txt_provider=None, logger=None):
-        self.vw_path = vw_path
-        self.runtime = runtime
-        self.job_pool = job_pool
-        self.logger = logger
-        self.cache_path_gen = cache_path_gen
-        self.model_path_gen = model_path_gen
-        self.pred_path_gen = pred_path_gen
-        self.cache_provider = cache_provider
-        self.txt_provider = txt_provider
+import os
+from DashboardMpi.helpers import logger, runtime, pool
+from DashboardMpi.helpers.input_provider import CachesProvider, LocalLogsProvider, ModelsProvider, PredictionsProvider
+
+
+class Environment:
```

Comment:
probably we should rename similarly all classes to be consistent - configuration / seq_pool / multiproc_pool / console_logger / ...